src/main/resources/reference.conf

```diff
@@ -1649,54 +1649,32 @@ resourceTypes = {
       read_policies = {
         description = "view all policies and policy details for this private azure storage account"
       }
-      identify = {
-        description = "use the identity that has access to this private azure storage account"
-      }
-      "share_policy::admin" = {
-        description = "change the membership of the admin policy for this private azure storage account"
-      }
-      "share_policy::user" = {
-        description = "change the membership of the user policy for this private azure storage account"
-      }
-    }
-    ownerRoleName = "admin"
-    roles = {
-      admin = {
-        roleActions = ["delete", "read_policies", "use", "share_policy::admin", "share_policy::user", "identify"]
-      }
-      user = {
-        roleActions = ["identify"]
-      }
-    }
-    allowLeaving = false
-    reuseIds = true
-  }
-
-  azure_managed_identity = {
-    actionPatterns = {
-      delete = {
-        description = "Delete this azure managed identity"
+      read = {
+        description = "read from the private azure storage account"
       }
-      read_policies = {
-        description = "view all policies and policy details for this azure managed identity"
+      write = {
+        description = "write to the private azure storage account"
       }
-      identify = {
-        description = "use the identity that has access to this azure managed identity"
+      "share_policy::owner" = {
+        description = "change the membership of the owner policy for this private azure storage account"
       }
-      "share_policy::admin" = {
-        description = "change the membership of the admin policy for this azure managed identity"
+      "share_policy::reader" = {
+        description = "change the membership of the reader policy for this private azure storage account"
       }
-      "share_policy::user" = {
-        description = "change the membership of the user policy for this azure managed identity"
+      "share_policy::writer" = {
+        description = "change the membership of the writer policy for this private azure storage account"
       }
     }
-    ownerRoleName = "admin"
+    ownerRoleName = "owner"
     roles = {
-      admin = {
-        roleActions = ["delete", "read_policies", "use", "share_policy::admin", "share_policy::user", "identify"]
+      owner = {
+        roleActions = ["delete", "read_policies", "share_policy::owner", "share_policy::reader", "share_policy::writer", "read", "write"]
```

Comment:
do they need pull_image?

Reply:
I think it makes sense that an `owner` has all the permissions of other roles.